src/selection.ts

```diff
@@ -0,0 +1,62 @@
+export enum SelectionTypes {
```

Comment:
FYI, we are now using string literals for this kind of thing instead.  

You can just do:

a) Simple type declaration 
```
export type SelectionType = 'single' | 'multi' | 'interval';
```

Or b) declare a namespace + type like @domoritz  did (e.g., for [ScaleType](https://github.com/vega/vega-lite/blob/master/src/scale.ts)).  


IMHO, I prefer a) because you can avoid importing types and constants when you use a function. 
(You still need to import the type for function declaration though.)